libcontainer/nsenter/nsexec.c

```diff
@@ -24,21 +22,21 @@
 
 /* All arguments should be above stack, because it grows down */
 struct clone_arg {
-	/*
-	 * Reserve some space for clone() to locate arguments
-	 * and retcode in this place
-	 */
-	char stack[4096] __attribute__ ((aligned(16)));
-	char stack_ptr[0];
-	jmp_buf *env;
+    /*
+     * Reserve some space for clone() to locate arguments
+     * and retcode in this place
+     */
+    char stack[4096] __attribute__ ((aligned(16)));
+    char stack_ptr[0];
+    jmp_buf *env;
```

Comment:
Just realized that I've used tab-width of 4 instead of 8, will fix sorry (although 7 seems to be what was used previously)